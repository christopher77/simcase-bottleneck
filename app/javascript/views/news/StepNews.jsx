import React from "react";
import styles from "./News/News.module.css";
import ImageSvg from "../../components/ui/ImageSvg";

const StepNews = props => {
  const { image, announce, report, title, finishMessage } = props;

  return (
    <div className={styles.newsContainer}>
      <ImageSvg src={image} />
      <div className={styles.contentContainer}>
        <h5>{title}</h5>
        <h5>{report}</h5>
        <h5>{announce}</h5>
        <h5>{finishMessage}</h5>
      </div>
    </div>
  );
};

export default StepNews;
