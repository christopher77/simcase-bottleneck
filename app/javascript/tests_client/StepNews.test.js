import React from "react";
import { render } from "@testing-library/react";
import StepNews from "../views/news/StepNews";

test("StepNews component", () => {
  const { asFragment } = render(<StepNews />);
  expect(asFragment).toMatchSnapshot();
});
