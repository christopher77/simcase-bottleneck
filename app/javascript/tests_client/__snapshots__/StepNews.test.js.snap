// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepNews component 1`] = `[Function]`;
